debian/rules

```diff
@@ -1,114 +1,25 @@
 #!/usr/bin/make -f
-# -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# GNU copyright 1997 to 1999 by Joey Hess.
 
-# Uncomment this to turn on verbose mode.
```

Comment:
I'd leave this comment untouched, because it comments the following line which you decided to leave.